portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,85 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body class="MainContainer">
+    <div class="Content">
+
+      <div class="Headline">
+        <div class="Name">Hi there, I'm Shradha </div>
+        <span class="Hand">👋</span> 
+      </div>
+      <br>
+      <img class="MyPic" src="/images/my_photo.jpeg" alt="My Picture."> 
+      <br><br>
+
+      <div class="AboutMeText">
+        <p class="Bio">
+          Born and raised in Haryana,
+          I am a final year Computer Engineering student from NSIT, Delhi.
+          My experience in tech includes working with Web Development, Linux OS and Machine Learning.
+          I also like working with neural networks and open source APIs.
+          In my college, I am a part of a technical society, taking classes for juniors to share my experience and knowledge of stuff like data structures, algorithms and web development.
+          Apart from that I have also been a member of the theatrical society to do stage plays.
+          For leisure, I enjoy watching a lot of TV series and movies.
+          <br>
+        </p>
+      </div>
+
+      <div>
+        <h1>Recent projects</h1>
+        <ul class="List">
+          <li>
+            <b> Twitter Bot Detection </b> - Machine Learning(RVM)
+            <button onclick="showProjectDetail(event)" id="1" tabindex="0">Details</button>
```

Comment:
nit: Ideally we also specify the `role` attribute of a button so that screen readers know what to read out to specially abled persons.